airbyte-integrations/connectors/destination-bigquery/src/test-integration/kotlin/io/airbyte/integrations/destination/bigquery/BigQueryDestinationTest.kt

```diff
@@ -490,6 +495,8 @@ internal class BigQueryDestinationTest {
             Path.of("secrets/credentials-no-edit-public-schema-role.json")
         protected val CREDENTIALS_WITH_GCS_STAGING_PATH: Path =
             Path.of("secrets/credentials-gcs-staging.json")
+        protected val CREDENTIALS_WITH_GCS_BAD_COPY_PERMISSION_PATH: Path =
+            Path.of("secrets/credentials-1s1t-gcs-bad-copy-permission.json")
```

Comment:
can you add something to `destination-bigquery/readme.md` explaining how you set up these credentials?

(also, checking my understanding: this config fails during the new `checkBigQueryCopyPermission`?)

Reply:
The readme.md is being used only for high level information about the project, so I have created a google doc with the steps that were used to create the service account used for testing

https://docs.google.com/document/d/16HhHOHJl_FDgi2UE14YtE-9B7rcOk9I9-gsmu7JUnRo/edit

Reply:
nice. it would be good to link to this doc from the source code then, so we can find it easily in the future

(I have no trust in google drive's search functionality :P )

Reply:
bump on adding a link in a comment, for better discoverability

Reply:
I checked the readme.md file, it has relevant content for setting up the environment etc, so I have added the steps for adding the required BigQuery permissions in the readme.md file